aioshelly/const.py

```diff
@@ -699,6 +700,13 @@ class ShellyDevice:
         gen=GEN2,
         supported=True,
     ),
+    MODEL_PRO_2_V3_UL: ShellyDevice(
+        model="SPSW-202XE12UL",
+        name="Shelly Pro 2",
```

Comment:
```suggestion
        name="Shelly Pro 2 UL",
```

I noticed that for the Plus we did specify in the name that the device is UL:
https://github.com/home-assistant-libs/aioshelly/blob/e60f9f0d009710042cf70d1527eedbe1096ce7aa/aioshelly/const.py#L537